### ISSUE TITLE
filter benchmarks: divisibleBy{X} variables does not correspond to the real operations that do

### ISSUE THREAD

**Issue by xgbuils** [benji6/imlazy#31]
Hi @benji6 

I was reading the benchmarks implementation and I found in `benchmarks/filter.js`:

``` javascript
const divisibleBy2 = x => x + 1
const divisibleBy3 = x => x / 2
const divisibleBy5 = x => 3 * x
```

It seems a copy past of `benchmarks/map.js`

Cheers!

**Comment by benji6** [comment]
Yep, you're right, that's what I've done. I've also noticed my performance section is all out of date since they updated the V8 engine in node. `imlazy` isn't as fast in the benchmarks anymore. I'll fix that typo and make some changes in the readme about performance

**Comment by benji6** [comment]
I've pushed some changes (e7521bb522932c177c4a1a79e77cb41d7fecd7f8). Thanks for pointing out this out @xgbuils 👍

**Comment by xgbuils** [comment]
You're welcome!

Ouch! I'm looking at your changes and I think that I don't express myself well. `filter` it's a function that usually receives a function of type `a -> Boolean` and you are passing a function like `Number -> Number` that could be more suitable for a `map` function. Then I expected:

``` javascript
const divisibleBy2 = x => x % 2 === 0
const divisibleBy3 = x => x % 3 === 0
const divisibleBy5 = x => x % 5 === 0
```

I mean, I think that the intent of the names was right. The problem was the definition.

Cheers!

**Comment by benji6** [comment]
Oh man - yeah sorry I see the mistake now! So embarrassing! I'll put something more reasonable in there.

Cheers
